docs/machine-learning/automate-training-with-model-builder.md

```diff
@@ -68,17 +59,21 @@ Regression is used to predict numbers.
 
 ![Diagram showing regression examples such as price prediction, sales forecasting, and predictive maintenance](media/regression-examples.png)
 
-Price prediction can be used to predict house prices using location, size, and other characteristics of the house. It is an example of a regression model type.
+Price prediction can be used to predict house prices using location, size, and other characteristics of the house. It is an example of the regression machine learning task.
 
 You can use the price prediction template for your scenario if you want to predict a numerical value with your own dataset.
 
-#### Custom scenario (choose your model type)
+#### Classify images into categories
 
-The custom scenario allows you to manually choose your model type.
+This scenario is a special case of multiclass classification, where the input data to be categorized is a set of images.
+
+#### Custom scenario
+
+The custom scenario allows you to manually choose your scenario.
 
 ## Data
 
-Once you have chosen your model type, Model Builder asks you to provide a dataset. The data is used to train, evaluate, and choose the best model for your scenario.
+Once you have chosen your scenario, Model Builder asks you to provide a dataset. The data is used to train, evaluate, and choose the best model for your scenario.
```

Comment:
Can we add file types we support here: 

From README:
"We currently support .tsv, .csv, .txt, and SQL as the data types you can use as datasets for Model Builder. If you have a .txt file, columns should be separated with ',' or ';' or '/t'. The files must have a header row."

We also now support jpg and png for image classification.

Reply:
You can also link to the load data how-to if the users want to learn more. https://docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/load-data-model-builder

Reply:
Good idea